NativeScript/runtime/js/inline-functions.js

```
@@ -7,6 +7,39 @@ const {
+        // Fresh object: ObjCExposedMethods may be inherited through the static
+        // chain, and assigning into it would edit the base class's.
+        target.ObjCExposedMethods = ObjectAssign({}, target.ObjCExposedMethods, options.exposedMethods);
+    }
+
+    if (options.eager) {
+        // Standard decorators run before static field initializers, so a class
+        // whose protocols/exposedMethods come from `static` fields would be
+        // registered without them. addInitializer defers to after the class is
+        // fully defined. Legacy decorators and plain calls already run late.
+        if (context && typeof context.addInitializer === 'function') {
+            context.addInitializer(function () { __registerNativeClass(this); });
```

ESLint

[error] '__registerNativeClass' is not defined.

(no-undef)